Sources/NIOHTTP2/HTTP2ChannelHandler.swift

```diff
@@ -1067,6 +1098,24 @@ extension NIOHTTP2Handler {
             return
         }
 
+        // Tell the delegate, if there is one.
+        if let delegate = self.frameDelegate {
+            switch frame.payload {
+            case .headers(let headers):
+                delegate.wroteHeaders(headers.headers, endStream: headers.endStream, streamID: frame.streamID)
+
+            case .data(let data):
+                switch data.data {
+                case .byteBuffer(let buffer):
+                    delegate.wroteData(buffer, endStream: data.endStream, streamID: frame.streamID)
+                case .fileRegion:
+                    ()
```

Comment:
Why do we take no action here? If it's intended should it be documented that we don't notify for file regions?